src/Http/Controllers/PropertyValueController.php

```diff
@@ -35,12 +35,15 @@ public function create(CreatePropertyValueForm $request, Property $property)
         ]);
     }
 
+    /**
+     * @refactored 07-07-2021 sd@groundwow.com
+     */
```

Comment:
Please remove @refactored comments from the PR

Reply:
Will do